### DISCUSSION TITLE
How to use java agent - plugin.http.include_http_headers

### DISCUSSION THREAD

**Discussion by yueguiji** [apache/skywalking#11335] [Q&A] [answered]
I want to collect value about json-payload(Http post)  in sprintMVC and show it with trace.
What should I do.
![image](https://github.com/apache/skywalking/assets/92442632/d17a7116-6656-429a-ace2-7a839539656e)

**Comment by yueguiji** [comment]
like this
![image](https://github.com/apache/skywalking/assets/92442632/ff22674a-b5bb-498b-b9a7-04f6cfdf6a3e)

**Comment by wu-sheng** [comment]
What do you mean? This config is about HTTP headers. Do you have any misunderstanding?

**Reply by yueguiji** [reply]
https://github.com/apache/skywalking/issues/5329
In this Task, I could see Payload In the Feature.So I think this headers could  include json-payload.

**Reply by wu-sheng** [reply]
The config asks you to set up the header key. Have you done that?
Your asking is out of context, I can't see the point.

**Reply by yueguiji** [reply]
OK，I know the json value in the request body , the http headers can not search.
So In my case, I want to collect the json value in the http post(SprintMVC - Content-Type:application/json).
What should I do.
![image](https://github.com/apache/skywalking/assets/92442632/caa79e98-2544-4472-b677-4cc1dab95317)

**Reply by wu-sheng** [reply, marked as answer]
I don't think we support collecting body. Only key=value pairs as parameter maps are supported.

**Reply by yueguiji** [reply]
OK Thanks for your answer.
